web/src/Button.tsx

```diff
@@ -2,13 +2,18 @@ import React from 'react'
 import './Button.css'
 
 interface ButtonIface {
-  onClick?: React.MouseEventHandler<HTMLAnchorElement>
+  onClick?: Function
```

Comment:
`(ButtonIface) => void`?

Reply:
i'm not sure i get you ?

Reply:
Function is too generic. Instead define the functions parameters and return type.